frontend/individuals.tsx

```diff
@@ -0,0 +1,96 @@
+import Remove from '@material-ui/icons/Remove';
+import SaveAlt from '@material-ui/icons/SaveAlt';
+import Search from '@material-ui/icons/Search';
+import ViewColumn from '@material-ui/icons/ViewColumn';
+
+import { makeStyles } from '@material-ui/core/styles';
+import { useDataContext } from './data_context'
+import { Individual } from '~data_context_global';
+
+// Define styles for tab menu
+const useStyles = makeStyles({
+  table: {
+    height: "calc(100% - 75px)",
+    padding: "0 20px",
+    overflowY: "scroll",
+  },
+});
+
+const tableIcons = {
+  Add: forwardRef((props, ref) => <AddBox {...props} ref={ref} />),
+  Check: forwardRef((props, ref) => <Check {...props} ref={ref} />),
+  Clear: forwardRef((props, ref) => <Clear {...props} ref={ref} />),
+  Delete: forwardRef((props, ref) => <DeleteOutline {...props} ref={ref} />),
+  DetailPanel: forwardRef((props, ref) => <ChevronRight {...props} ref={ref} />),
+  Edit: forwardRef((props, ref) => <Edit {...props} ref={ref} />),
+  Export: forwardRef((props, ref) => <SaveAlt {...props} ref={ref} />),
+  Filter: forwardRef((props, ref) => <FilterList {...props} ref={ref} />),
+  FirstPage: forwardRef((props, ref) => <FirstPage {...props} ref={ref} />),
+  LastPage: forwardRef((props, ref) => <LastPage {...props} ref={ref} />),
+  NextPage: forwardRef((props, ref) => <ChevronRight {...props} ref={ref} />),
+  PreviousPage: forwardRef((props, ref) => <ChevronLeft {...props} ref={ref} />),
+  ResetSearch: forwardRef((props, ref) => <Clear {...props} ref={ref} />),
+  Search: forwardRef((props, ref) => <Search {...props} ref={ref} />),
+  SortArrow: forwardRef((props, ref) => <ArrowDownward {...props} ref={ref} />),
+  ThirdStateCheck: forwardRef((props, ref) => <Remove {...props} ref={ref} />),
+  ViewColumn: forwardRef((props, ref) => <ViewColumn {...props} ref={ref} />)
+};
+
+const columns = [
+  {field: 'herd', title: 'Besättning'},
+  {field: 'name', title: 'Namn'},
+  {field: 'certificate', title: 'Certifikat'},
+  {field: 'number', title: 'Nummer'},
+  {field: 'sex', title: 'Kön'},
+  {field: 'birthDate', title: 'Födelsedatum'},
+  {field: 'mother', title: 'Moder'},
+  {field: 'father', title: 'Fader'},
+  {field: 'color', title: 'Färg'},
+  {field: 'colorNote', title: 'Färganteckning'},
+]
+
+/**
+ * Shows a list of all genebanks, with links to the individual genebanks.
+ */
+export function IndividualsTable({id}: {field: number}) {
+  const [individuals, setIndividuals] = React.useState([] as Array<Individual>)
+  const {genebanks} = useDataContext()
+  const classes = useStyles();
+
+  React.useEffect(() => {
+    let genebank = genebanks.filter(g => g.id == id)
+    if (genebank) {
+      setIndividuals(genebank[0].individuals)
+    }
+  }, [genebanks])
+
+  return <>
+    <div className={classes.table}>
+      <MaterialTable
+        icons={tableIcons}
+        columns={columns}
+        data={individuals}
+        title="Alla individer"
+      />
```

Comment:
Moder och Fader seems to refer to some internal id (like 11793) while Nummer for a given bunny is written in another form (like G1390-1911).

Reply:
Alright!